community/xclip/version

```diff
@@ -0,0 +1 @@
+0.13
```

Comment:
Missing `rel`.

Reply:
(`0.13 1`)

Reply:
I forgot about that, sorry!
Seems to be fixed now.